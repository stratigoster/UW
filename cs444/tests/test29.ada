PACKAGE foo IS
  type coins is (1,5,10,25);
--               ^
-- identifier expected, but found numeric literal
END foo;
