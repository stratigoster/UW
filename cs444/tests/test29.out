***ERROR***:2:18:	Expected "Id" but found "1"
